components/PageHeader.vue

```diff
@@ -58,6 +75,69 @@
         type: Boolean,
         default: false
       }
+    },
+
+    data() {
+      return {
+        scrollPosition: 0,
+        windowWidth: 992,
```

Comment:
What's the reason for this being set to 992? Could it be null or 0?

Reply:
it could be 0/null but then handleScroll function would not run on load.
windowWidth is set dynamically after mounted() runs, so it may be fine without it, just thought it was safer this way

Reply:
have tested setting to 0 and since this.getWindowWidth(); is called within mounted there's no issue